.github/workflows/ci.yaml

```
@@ -40,7 +45,7 @@ jobs:
         key: $clippy-cache-${{ steps.toolchain.outputs.rustc_hash }}-${{ hashFiles('**/Cargo.lock') }}
 
     - name: "Clippy ${{ matrix.rust }}"
-      run: cargo clippy --features=rt-tokio-crypto-rust --all-targets --all -- -D clippy::dbg_macro -D warnings -F unused_must_use
+      run: cargo clippy --features=${{ feature }} --all-targets --all -- -D clippy::dbg_macro -D warnings -F unused_must_use

```

You'll need to use `matrix.feature` here (and in the other places) instead, like what is already done for `matrix.rust` in the toolchain setup:
```suggestion
      run: cargo clippy --features=${{ matrix.feature }} --all-targets --all -- -D clippy::dbg_macro -D warnings -F unused_must_use

```

---

Fixed, sorry about that.

---

No problem!